framework/rendering/render_context.cpp

```diff
@@ -285,6 +291,31 @@ CommandBuffer &RenderContext::begin(CommandBuffer::ResetMode reset_mode)
 		throw std::runtime_error("Couldn't begin frame");
 	}
 
+	static VkExtent2D previous1;
```

Comment:
Is there are more consistent way (in terms of how the framework is currently working) of doing this without static variables? If not, this code should have some comments to explain how this works and why this code is in place.

Reply:
I'm sorry for not making myself clear.

This commit was just for logging the swapchain extent for the issue https://github.com/KhronosGroup/Vulkan-Samples/issues/337#issuecomment-916984745.

The suggested commit is #c5b957a79715020054a4a81ecfd6bc285dc50d54.